app/services/zendesk_cleaner.rb

```diff
@@ -0,0 +1,27 @@
+class ZendeskCleaner
+  STAFF_INBOX = 'staff.prison.visits created'.freeze
+
+  def initialize(zendesk_client)
```

Comment:
I think including  the `ZendeskClient`  module should do the trick rather than including it in the rake task